osf/models/node.py

```diff
@@ -1665,6 +1665,7 @@ def fork_node(self, auth, title=None, parent=None):
             addon.after_fork(original, forked, user)
 
         forked.save()
+        update_storage_usage_cache(forked.id, forked._id)  # Files were copied in the addons. Need to update storage usage
```

Comment:
If the cache is updating on first save why is this necessary? Shouldn't it update the cache on the previous line?

Reply:
This isn't the first save. L1595 is the first save.

Reply:
Oh I see now, the hooks obviously aren't firing here, because it's just metadata.